status-im/status-legacy
# Restrict amount of characters in typed message

fixes TBD

### Summary

Cap max characters count in a message to 4096. (As suggested here - https://github.com/status-im/security-reports/issues/22#issuecomment-674998679)

### Review notes
This is a partial solution of a long-messages problem. Another part will be dropping long messages inside `status-go` as @cammellos  suggested.

#### Platforms
- Android
- iOS

#### Areas that maybe impacted
Text input in chats


### Steps to test
- Open Status
- open any chat 
- Make sure user can't input more than 4096 characters

status: ready
